services/encoding_service.go

```diff
@@ -389,6 +389,80 @@ func (s *EncodingService) RetrieveMP4MuxingCustomData(encodingID string, mp4ID s
 	return &r, nil
 }
 
+func (s *EncodingService) AddProgressiveMOVMuxing(encodingID string, a *models.ProgressiveMOVMuxing) (*models.ProgressiveMOVMuxingResponse, error) {
+	b, err := json.Marshal(*a)
+	if err != nil {
+		return nil, err
+	}
+	path := EncodingEndpoint + "/" + encodingID + "/" + "muxings/progressive-mov"
+	o, err := s.RestService.Create(path, b)
+	if err != nil {
+		return nil, err
+	}
+	var r models.ProgressiveMOVMuxingResponse
+	err = json.Unmarshal(o, &r)
+	if err != nil {
+		return nil, err
+	}
+	return &r, nil
+}
+
+func (s *EncodingService) RetrieveProgressiveMOVMuxing(encodingID string, progressiveMOVID string) (*models.ProgressiveMOVMuxingResponse, error) {
+	path := EncodingEndpoint + "/" + encodingID + "/" + "muxings/progressive-mov" + "/" + progressiveMOVID
```

Comment:
wouldn't it be easier to do `..+ "muxings/progressive-mov/" + progressiveMOVID` instead of `...+ "muxings/progressive-mov" + "/" + progressiveMOVID`?

Reply:
Yeah.  This might have been some copy pasta where everything else following that scheme

Reply:
I'll leave the minor refactoring for a future PR.